gulpfile.js

```diff
@@ -13,7 +12,7 @@ const gulpif = require('gulp-if');
 const gutil = require('gulp-util');
 const imagemin = require('gulp-imagemin');
 const minifycss = require('gulp-minify-css');
-const neat = require('node-neat');
+const normalize = require('node-normalize-scss');
```

Comment:
Now that we're using **node-normalize-scss**, I'm not sure if we need the `npm:normalize` Gulp task below.